primefaces/src/main/java/org/primefaces/renderkit/OutcomeTargetRenderer.java

```diff
@@ -164,7 +161,7 @@ protected String getTargetURL(FacesContext context, UIOutcomeTarget outcomeTarge
                 clientWindow = context.getExternalContext().getClientWindow();
 
                 if (clientWindow != null) {
-                    clientWindowRenderingModeEnabled = ((ClientWindow) clientWindow).isClientWindowRenderModeEnabled(context);
+                    boolean clientWindowRenderingModeEnabled = ((ClientWindow) clientWindow).isClientWindowRenderModeEnabled(context);
```

Comment:
i think this break nested enabled/disabled, therefore i saved the old value

Reply:
Ok I see, I reverted but removed clientWindow unecessary check (just SONAR like pointed out)